docs/source/2020CAPE.rst

```diff
@@ -0,0 +1,149 @@
+.. 2020CAPE documentation master file, created by
+   sphinx-quickstart on Mon Jul  6 13:31:15 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+
+.. _2020 July CAPE Case:
+2020 July CAPE Case
+=====================================
+
+This case illustrates the low Convective Available Potential Energy (CAPE) issue during summertime in GFS.v16 identified by NOAA Environmental Modeling Center (EMC) Model Evaluation Group (MEG).
+
+..............................
+Model Configuration and Datasets
+..............................
+.. tabs::
+  .. group-tab:: MRW.v1.0
+
+    The case runs are initialized at 00z Jul 23, 2020 with 24 hours forecasting. The corresponding namelist options that need to be changed are listed below. The app uses ``./xmlchange`` to change the runtime settings. The settings that need to be modified to set up the start date, start time, and run time are listed below.
```

Comment:
24 hours forecast length ?

Reply:
Sorry, I was referring to the language. Please use `with a 24h forecast length` instead of `with 24 hours forecasting`.

Reply:
Ah, right! Fixed now. Thank you Dom!